src/nextjournal/clerk/render/editor.cljs

```diff
@@ -21,10 +21,10 @@
             [shadow.esm]))
 
 (defn eval-string
-  ([source] (sci/eval-string* (sci.ctx-store/get-ctx) source))
+  ([source] (eval-string (sci.ctx-store/get-ctx) source))
   ([ctx source]
    (when-some [code (not-empty (str/trim source))]
-     (try {:result (sci/eval-string* ctx code)}
+     (try (sci/eval-string* ctx code)
```

Comment:
@borkdude not sure what's going on here. This change just adds the try/catch for arity-1, but the issue persists. Can you see why ns aliases from evaluating ns forms don't stick in the context?